api/tests/test_block_processor.py

```diff
@@ -240,6 +235,111 @@ async def main():
+            RetrievedTokenTransfer(transactionHash='0x701fdfc3b447945304f61e5ff3117a17ad2bbcb90eb447239d710b925a37600c', registryAddress='0x9b8b9c7C02be0bD0AA4D669bF6a1f6003424c6Dc', tokenId='7030', fromAddress='0x7701cC2986207232b88e88DFDd4E1BE18B5381b9', toAddress='0x0E1b9E1C3E0d3a7dB5E5980b95E22F3814CB2Aa7', operatorAddress='0x0E1b9E1C3E0d3a7dB5E5980b95E22F3814CB2Aa7', amount=1, value=20000000000000000, gasLimit=336344, gasPrice=21536358794, blockNumber=14806164, tokenType='erc721'),
+            RetrievedTokenTransfer(transactionHash='0x260dd1d3f73667e763d8884048919f7190af2d153ec395455f6f66fece2c6c20', registryAddress='0x3F93d710Ff8c449eA300AD69412E7B8F289a2954', tokenId='5118', fromAddress='0x5eC3AF4a9DC4A19Ac3b4828ecB390422D4347B6d', toAddress='0x837253Fa2D3f43A07Dd8a8CE2c081AaD8f670d5d', operatorAddress='0x837253Fa2D3f43A07Dd8a8CE2c081AaD8f670d5d', amount=1, value=158000000000000000, gasLimit=362407, gasPrice=21536358794, blockNumber=14806164, tokenType='erc721'),
+            RetrievedTokenTransfer(transactionHash='0xa189b59a102ccf3937989ec421604a3d04d7a53c175cf88f9c8605346fc434c2', registryAddress='0x4C1f84b2780ebd995d2E536b640cCA14586298A2', tokenId='1', fromAddress='0x1524C4619333bC15Bb5E340E506E5B9cB8D62223', toAddress='0xcDA72070E455bb31C7690a170224Ce43623d0B6f', operatorAddress='0x1524C4619333bC15Bb5E340E506E5B9cB8D62223', amount=1, value=0, gasLimit=231242, gasPrice=21536358794, blockNumber=14806164, tokenType='erc721'),
+            RetrievedTokenTransfer(transactionHash='0xf03328c4603410549560b6f44be742034d8cda763f8178aa171038d425086c88', registryAddress='0x883E9ef3391A43dAbA2dff04b64b1F2921E55C1A', tokenId='2727', fromAddress='0xf0D2a721ca6F5852f07A75A330CE83Ac92053fEb', toAddress='0x3ee017a754E24148dB5b8144E1B46Cf81c691147', operatorAddress='0x3ee017a754E24148dB5b8144E1B46Cf81c691147', amount=1, value=18800000000000000, gasLimit=307447, gasPrice=21446358794, blockNumber=14806164, tokenType='erc721'),
+            RetrievedTokenTransfer(transactionHash='0x466e880275ee3f815d0c0bd787009632a437eefe1e7e7dab753d21ea16c0bd8b', registryAddress='0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D', tokenId='7963', fromAddress='0x13d8faF4A690f5AE52E2D2C52938d1167057B9af', toAddress='0xAf6209795446f33492133dF966042BC2B568d866', operatorAddress='0xAf6209795446f33492133dF966042BC2B568d866', amount=1, value=0, gasLimit=515008, gasPrice=21442773112, blockNumber=14806164, tokenType='erc721'),
+            RetrievedTokenTransfer(transactionHash='0x466e880275ee3f815d0c0bd787009632a437eefe1e7e7dab753d21ea16c0bd8b', registryAddress='0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D', tokenId='7139', fromAddress='0xAf6209795446f33492133dF966042BC2B568d866', toAddress='0x0232d1083E970F0c78f56202b9A666B526FA379F', operatorAddress='0xAf6209795446f33492133dF966042BC2B568d866', amount=1, value=7005000000000000000, gasLimit=515008, gasPrice=21442773112, blockNumber=14806164, tokenType='erc721'),
```

Comment:
transaction hash 0x466e880275ee3f815d0c0bd787009632a437eefe1e7e7dab753d21ea16c0bd8b should return 0 for transfer 7963 and 7ether for transfer of 7138